server/model/newExamResultsSubscriber.go

```diff
@@ -0,0 +1,13 @@
+package model
+
+import (
+	"database/sql"
+	"time"
+)
+
+type NewExamResultsSubscriber struct {
+	CallbackUrl    string         `gorm:"primary_key" json:"callbackUrl"`
+	ApiKey         sql.NullString `json:"-"`
+	CreatedAt      time.Time      `gorm:"autoCreateTime" json:"createdAt"`
+	LastNotifiedAt sql.NullTime   `json:"lastNotifiedAt"`
```

Comment:
We switched to only using `null` instead of mixing the two.

```suggestion
	LastNotifiedAt null.Time   `json:"lastNotifiedAt"`
```